### PR TITLE
Task-iframes completed

### PR DESCRIPTION
**Pull request by G4GANCHAUDHARY**
Web capture_19-11-2021_3451_127 0 0 1

### CODE REVIEW

File: src/Assignment-3/assignment3.html [hunk 1]
```diff
@@ -12,6 +12,24 @@
...
+		<div id="i1"class="iframe">
+			<iframe src="https://en.wikipedia.org/wiki/Wikipedia" id="render-Wiki" title="wikipedia"></iframe>
+		</div>
+	</div>
+	<div id="container2" class="bigcontainer">
+	   <h1 class="headerline">Your Best Value Proposition</h1>
+	   <p class="para">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua</p>
+	   <button class="butt">Start free trial</button>
+	</div>
+	<div id="container3" class="bigcontainer">
```
Comment (Tarunasudani) [inline src/Assignment-3/assignment3.html:30]: this container 2 and container 3 should be bounded inside one single container

**Review by Tarunasudani — COMMENTED** [review]
2 containers needs to be merged

**Comment by Sanchitanwar0007** [comment]
Everything looks good and fine. The webpage looks same as given in the assignment.

File: src/Assignment-3/assignment3.html (hunk 1 above)
Comment (G4GANCHAUDHARY) [reply src/Assignment-3/assignment3.html:30]: It was mentioned to create three separate containers of full width.

**Comment by G4GANCHAUDHARY** [comment]
> 2 containers needs to be merged

It was mentioned to create three separate containers of full width.

**Review by Sanchitanwar0007 — APPROVED** [review]

**Pull request merged**